configdb/hardware/models.py

```diff
@@ -12,21 +12,27 @@ class Meta:
 
 
 class Site(BaseModel):
-    active = models.BooleanField(default=True)
-    code = models.CharField(max_length=3)
-    name = models.CharField(default='', blank=True, max_length=200)
-    lat = models.FloatField(default=0.0)
-    long = models.FloatField(default=0.0)
-    elevation = models.IntegerField(help_text='meters')
+    active = models.BooleanField(default=True, help_text='Whether the site is active and able to accept observations')
+    code = models.CharField(max_length=3, help_text='3-letter site code')
+    name = models.CharField(default='', blank=True, max_length=200, help_text='Site name')
+    lat = models.FloatField(default=0.0, help_text='Site latitude in decimal degrees')
+    long = models.FloatField(default=0.0, help_text='Site longitude in decimal degrees')
+    elevation = models.IntegerField(
+        validators=[MinValueValidator(0), MaxValueValidator(100000)], 
```

Comment:
Made this 0-100,000. Can change it when we put a telescope on the moon, I guess?

Reply:
This might not even be worth saying since generally telescopes are placed at higher elevation... but technically there are places on Earth that are below 0 meters in elevation https://en.wikipedia.org/wiki/List_of_places_on_land_with_elevations_below_sea_level The lowest is the Dead Sea, at about -430 meters.

Reply:
Some day I'll take a telescope + OCS to badwater basin in death valley (-86m) 😉 https://en.wikipedia.org/wiki/Death_Valley

Maybe we should give some wiggle room, just to cover the extents of the terrestrial earth